.github/workflows/ci.yml

```diff
@@ -207,7 +224,7 @@ jobs:
 
   status-check:
     name: Status Check
-    needs: [format, clippy, web-check, build, integration-tests, runBenchmark, runMemoryProfile]
+    needs: [format, clippy, web-check, build, format_qsc, integration-tests, runBenchmark, runMemoryProfile]
```

Comment:
Nittiest of nits: can you use a dash instead of underscore in the name to match the style of the other job names?

(And as I say this, I realize some of them use camelCase so we're already inconsistent :/ )

Reply:
done